optima/results.py

```diff
@@ -833,21 +874,34 @@ def diff(self, base=None):
                 resultsobj.pops[simkey] -= basepop
         
         return resultsdiff
-    
-    
-    def export(self, filename=None, folder=None, ind=None, writetofile=True, verbose=2, asexcel=True, **kwargs):
-        ''' A method to export each multiresult to a different sheet in Excel (or to a single large text file) '''
+
+    def export(self, filename=None, folder=None, ind=None, writetofile=True, verbose=2, asexcel=True,
+               comparisontab=False, **kwargs):
+        """ A method to export each multiresult to a different sheet in Excel (or to a single large text file) """
         
-        if asexcel: outputdict = odict()
-        else:       outputstr = ''
+        if asexcel:
+            outputdict = odict()
+        else:
+            outputstr = ''
         for key in self.keys:
-            thisoutput = Resultset.export(self, ind=ind, key=key, writetofile=False, **kwargs)
+            thisoutput = Resultset.export(self, ind=ind, key=key, writetofile=False, comparisontab=comparisontab,
+                                          **kwargs)
             if asexcel:
                 outputdict[key] = thisoutput
             else:
                 outputstr += '%s\n\n' % key
                 outputstr += thisoutput
                 outputstr += '\n'*5
+
+        if comparisontab:
+            thisoutput = Resultset.comparebudgets(self)
+            if asexcel:
+                outputdict['Comparison'] = thisoutput
+            else:
+                outputstr += 'Comparison\n\n'
+                outputstr += thisoutput
+                outputstr += '\n' * 5
+
```

Comment:
Have you tested that this does still work when writing to CSV instead of Excel?

Reply:
Good point, it doesn't, so this code can only execute when writing to excel, as well as being an optimization multiresultset.